### ISSUE TITLE
Don't forget..

### ISSUE THREAD

**Issue by michaelpittino** [upend/IF_MS_BUYS_GITHUB_IMMA_OUT#116]
![xkcd](https://imgs.xkcd.com/comics/free_speech.png)

**Comment by vassudanagunta** [comment]
Thank you. I'm in favor of you posting it on Issue 114.

**Comment by vassudanagunta** [comment]
Hello,

The GitHub Evacuation Project has moved to GitLab (not an endorsement or even a final home). Your enthusiasm and contribution is still needed. Please check out [the new project home](https://gitlab.com/upend/github), and read [the project wiki](https://gitlab.com/upend/github/evac-guide/wikis) for info to get restarted.

Thank you!

vas

**Comment by dominichayesferen** [comment]
@vassudanagunta Shall you archive this repository then so that it's locked on these issue messages? (prevents spam getting in the way of your announcement, just saying) (Settings > Dangerous Stuff > Archive This Repository)

**Comment by vassudanagunta** [comment]
@feren I never thought of this. Great idea!  If spam starts happening again, that is what I'll do. But for now, I don't want to prevent any constructive communication, though I'd prefer that happen at the project's [new (temporary) home](https://gitlab.com/upend/github).

**Comment by ghost** [comment]
Correct. What the first amendment does shield everyone from are people (regardless of whether they are the majority or minority, government or private, etc.) acting like total reprobates and throwing chaotic tantrums every single time someone they don't want to speak is given the platform to speak. What the first amendment protects everyone from is every citizen's right to express themselves being coercively torn away by a mob through means of force.

**Comment by vassudanagunta** [comment]
@citizen163 since you've evacuated GitHub, how about tagging your profile and swapping pics per the instructions in the README? I'm going to get the ball rolling on a campaign next week...
